packages/tables/src/examples/advanced-table.md

```diff
@@ -15,52 +15,54 @@ const StyledCaption = styled(Caption)`
   <StyledCaption>Your Unsolved Tickets</StyledCaption>
   <Head>
     <Row header>
-      <HeaderCell scope="col">Subject</HeaderCell>
-      <HeaderCell scope="col">Requester</HeaderCell>
-      <HeaderCell scope="col">Requested</HeaderCell>
-      <HeaderCell scope="col">Type</HeaderCell>
+      <HeaderCell width="25%">Subject</HeaderCell>
+      <HeaderCell width="25%">Requester</HeaderCell>
+      <HeaderCell width="25%">Requested</HeaderCell>
+      <HeaderCell width="25%">Type</HeaderCell>
     </Row>
   </Head>
   <Body>
     <Row group>
-      <Cell colSpan={4}>
+      <Cell width="100%">
```

Comment:
From our offline discussion... I've now split the Header and Group styles of Row into their own respective components: `HeaderRow` and `GroupRow`. This helps with the styling as well as conflicting prop-types since there shouldn't be focus/hover/selected, etc. styling on these combinations.